x/liquidfarming/keeper/grpc_query.go

```diff
@@ -270,11 +283,11 @@ func (k Querier) ExchangeRate(c context.Context, req *types.QueryExchangeRateReq
 		}
 
 		// MintRate = LFCoinTotalSupply / LPCoinTotalFarmingAmount
-		res.MintRate = lfCoinTotalSupplyAmt.ToDec().Quo(farm.TotalFarmingAmount.ToDec())
+		res.MintRate = lfCoinSupplyAmt.ToDec().Quo(position.FarmingAmount.ToDec())
 
 		// BurnRate = LPCoinTotalFarmingAmount - CompoundingRewards / LFCoinTotalSupply
```

Comment:
We need () like `// BurnRate = (LPCoinTotalFarmingAmount - CompoundingRewards) / LFCoinTotalSupply`